R/README.md

```diff
@@ -34,7 +34,7 @@
 
 ### Optional
 1. The codelabs load results of previously run Google Cloud Dataflow and Apache Spark jobs.
-  * If you would like to run the jobs yourself, please see [Getting Started with Dataflow](https://github.com/googlegenomics/dataflow-examples) and/or [Getting Started with Spark](https://github.com/googlegenomics/spark-examples).
```

Comment:
@mbookman do you think this should be updated to http://googlegenomics.readthedocs.org/en/latest/use_cases/compute_principal_coordinate_analysis/index.html  now that we have it?